### PR TITLE
Fix wrongful description of transfer-ownership

### PR DESCRIPTION
**Pull request by wiswedel**
clarify share handling

Signed-off-by: Sascha Wiswedel <sascha.wiswedel@nextcloud.com>

fixes https://github.com/nextcloud/documentation/issues/5451

### CODE REVIEW

File: admin_manual/configuration_server/occ_command.rst [hunk 1]
```diff
@@ -661,8 +661,11 @@ File operations
  files
   files:cleanup              cleanup filecache
   files:scan                 rescan filesystem
-  files:transfer-ownership   All files and folders are moved to another 
-                             user - shares are moved as well.
+  files:transfer-ownership   All files' and folders' ownerships are moved to another 
+                             user. Outgoing shares are moved as well.
+                             Incoming shares are not moved because the sharing user 
+                             holds the ownership of the respective files and therefore 
+                             owns the decision which actual persons to be shared with.
```
Comment (acsfer) [inline admin_manual/configuration_server/occ_command.rst:668]: Maybe replace **persons** with **users** to be consistent? Not a big deal :)

Comment (wiswedel) [reply admin_manual/configuration_server/occ_command.rst]: yah, fair enough. Persons is more logical, going down to the intention that swings behind the reason of this behaviour, but users is actually more common.
Fixed it.

**Review by acsfer — APPROVED** [review]

**Pull request merged**